src/main/java/io/gravitee/policy/requestvalidation/RequestValidationPolicy.java

```diff
@@ -149,7 +149,9 @@ private Set<ConstraintViolation> validate(ExecutionContext executionContext) {
             TemplateEngine templateEngine = executionContext.getTemplateEngine();
             String input = null;
             try {
-                input = templateEngine.eval(rule.getInput(), String.class).blockingGet();
+                // TODO: Use the non deprecated method, but it requires APIM 3.20+ to have an up to date version of the `gravitee-node`
+                // to ensure the correct version of `gravitee-expression-language` + `rxjava` are provided.
+                input = templateEngine.getValue(rule.getInput(), String.class);
```

Comment:
Sounds better indeed 👍 
I know there is also a deprecated on the getValue that indicates to prefer the use of reactive eval but we have identified that the blocking eval should be kept. The idea is to have an `evalNow` method at the end (see https://gravitee.atlassian.net/browse/APIM-548)